execution/distinct.go

```diff
@@ -13,13 +13,18 @@ type Distinct struct {
 	storage        storage.Storage
 	source         Node
 	eventTimeField octosql.VariableName
+
+	garbageCollectorBoundary int // number of minutes subtracted from watermark to create boundary for garbage collection
+	garbageCollectorCycle    int // number of miliseconds for garbage collector to sleep between records check
```

Comment:
Is there a reason why the `garbageCollectorBoundary` is in minutes and `garbageCollectorCycle` in miliseconds? I think the most natural for both would be to store seconds